src/manager.hpp

```diff
@@ -183,6 +206,38 @@ class SessionManager final :
      *        and cleanup sessions of an unavailable service.
      */
     void checkSessionOwnerAlive(const boost::system::error_code&);
+
+    /**
+     * @brief isAllSessionsAllowed check if all sessions are allowed by
+     * specified ID.
```

Comment:
Indents 
     * @brief isAllSessionsAllowed check if all sessions are allowed by
     *        specified ID.
     * 
     * @param sessionId        - unique session ID to remove from storage
     *
     * @return bool true/false - is closing all sessions allowed